### PR TITLE
[new release] trampoline (0.1.0)

### PR DESCRIPTION
**Pull request by QAhell**
A trampoline implementation

- Project page: <a href="https://github.com/QAhell/trampoline_ocaml">https://github.com/QAhell/trampoline_ocaml</a>

##### CHANGES:

- Working trampoline implementation

### CODE REVIEW

File: packages/trampoline/trampoline.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "A trampoline implementation"
+description: "A trampoline implementationtion"
```
Comment (mseri) [inline packages/trampoline/trampoline.0.1.0/opam:3]: Can you make the `synopsis` and the `description` more descriptive?

Comment (QAhell) [reply packages/trampoline/trampoline.0.1.0/opam:3]: I'm working on it.

Comment (QAhell) [reply packages/trampoline/trampoline.0.1.0/opam]: @mseri The ocaml ci page seems to be unavailable. Did my test crash the test server?

Comment (avsm) [reply packages/trampoline/trampoline.0.1.0/opam]: @QAhell there was an unrelated hardware failure, not your fault at all! :-) https://github.com/ocaml/infrastructure/issues/51

Just restarted some failed CI jobs and should be ready to merge this package soon.

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Comment by QAhell** [comment]
Thank you for merging the pull request! I'll have a look at the discussion site when time permits.

**Pull request merged**